exporters/logging-otlp/src/test/java/io/opentelemetry/exporter/logging/otlp/TestDataExporter.java

```diff
@@ -48,14 +47,13 @@ abstract class TestDataExporter<T> {
       Resource.create(Attributes.builder().put("key", "value").build());
 
   private static final LogRecordData LOG1 =
-      TestExtendedLogRecordData.builder()
+      TestLogRecordData.builder()
           .setResource(RESOURCE)
           .setInstrumentationScopeInfo(
               InstrumentationScopeInfo.builder("instrumentation")
                   .setVersion("1")
                   .setAttributes(Attributes.builder().put("key", "value").build())
                   .build())
-          .setEventName("event name")
```

Comment:
Do these get covered somewhere else? Again I'm a little surprised to see the event names going away in tests.

Reply:
They would need to live in a new `testIncubating` module. I was lazy here and omitted since there is already coverage for this in `:exporters:otlp:common:testIncubating`, and `logging-otlp` depends on `:exporters:otlp:common`. 

Testing of experimental features in the main test suite was an oversight.